src/app/components/PriceChart.tsx

```diff
@@ -260,12 +262,75 @@ function PriceChartCanvas(props: PriceChartProps) {
   );
 }
 
-export function PriceChart() {
+enum ChartTabOptions {
+  TRADING_CHART = "TRADING_CHART",
+  PAIR_INFO = "PAIR_INFO",
+}
+
+export function TradingChartOrPairInfo() {
   const t = useTranslations();
-  const state = useAppSelector((state) => state.priceChart);
   const dispatch = useAppDispatch();
   const candlePeriod = useAppSelector((state) => state.priceChart.candlePeriod);
 
+  // Set default candlePeriod state as defined in initialState of priceChartSlice
+  useEffect(() => {
+    dispatch(setCandlePeriod(initialPriceChartState.candlePeriod));
+  }, [dispatch]);
+
+  const [currentTab, setCurrentTab] = useState(ChartTabOptions.TRADING_CHART);
+
+  return (
+    <div>
+      <div className="flex flex-col sm:flex-row justify-between sm:pr-10 pr-4 border-b-[0.5px] border-b-[rgba(255,255,255,0.1)]">
+        <div className="flex space-x-4 sm:space-x-5 pb-0 pt-2">
+          {[
+            [t("trading_chart"), ChartTabOptions.TRADING_CHART],
+            [t("pair_info"), ChartTabOptions.PAIR_INFO],
+          ].map(([title, tab], indx) => {
+            const isActive = tab === currentTab;
+            return (
+              <span
+                key={indx}
+                className={`text-sm sm:text-base pb-2 sm:pb-3 px-2 ${
```

Comment:
Keep the `text-base` class on mobile for consistency, as the "Order book" and "Recent trades" tabs also maintain the same font size on mobile.
Suggested edit:
```suggestion
                className={`text-base pb-2 sm:pb-3 px-2 ${
```